### ISSUE TITLE
Add sysdb unit tests for alias handling

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2014]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2010
- Created at 2013-07-09 14:18:52 by [sgallagh](https://pagure.io/user/sgallagh)
- Closed at 2020-03-24 14:18:24 as wontfix
- Assigned to [mzidek](https://pagure.io/user/mzidek)
---

Because RFC2307 supports aliases for usernames and group names with multivalued attributes, SSSD handles the non-RDN ones as extra names in the sysdb. We need to add a set of tests to the sysdb test suite to ensure that these aliases are properly handled during both case-sensitive and case-insensitive operation.

### Comments

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2013-07-11 16:02:08](https://pagure.io/SSSD/sssd/issue/2010#238095)**

Fields changed

milestone: NEEDS_TRIAGE => Interim Bucket
rhbz:  => 0

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2013-07-30 12:54:43](https://pagure.io/SSSD/sssd/issue/2010#238096)**

Fields changed

milestone: Interim Bucket => SSSD 1.12 beta

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-02-19 11:54:15](https://pagure.io/SSSD/sssd/issue/2010#238097)**

Fields changed

blocking:  => #1923

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-05-30 15:53:43](https://pagure.io/SSSD/sssd/issue/2010#238098)**

Fields changed

milestone: SSSD 1.12 beta => SSSD 1.12.1

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-09-08 20:09:06](https://pagure.io/SSSD/sssd/issue/2010#238099)**

Mass-moving all tickets that didn't make 1.12.1 into 1.12.2

milestone: SSSD 1.12.1 => SSSD 1.12.2

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-09-30 19:06:21](https://pagure.io/SSSD/sssd/issue/2010#238100)**

We need to do a release as requested by downstream. Moving tickets that are not fixed already or very close to acking to 1.12.3

milestone: SSSD 1.12.2 => SSSD 1.12.3

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-11-13 19:20:40](https://pagure.io/SSSD/sssd/issue/2010#238101)**

Fields changed

mark:  => 0
owner: somebody => mzidek

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2014-11-19 17:38:04](https://pagure.io/SSSD/sssd/issue/2010#238102)**

It makes sense to write the unit tests now instead of moving them to a later release again, but the tickets in 1.12.3 should be more focused at this point. Moving to 1.12.4

milestone: SSSD 1.12.3 => SSSD 1.12.4

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2015-02-12 20:05:32](https://pagure.io/SSSD/sssd/issue/2010#238103)**

Fields changed

milestone: SSSD 1.12.4 => SSSD Deferred

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2017-02-24 14:57:34](https://pagure.io/SSSD/sssd/issue/2010#238104)**

**Metadata Update from @sgallagh**:
- Issue assigned to mzidek
- Issue marked as blocked by: #1923
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2020-03-24 14:18:23](https://pagure.io/SSSD/sssd/issue/2010#636892)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2020-03-24 14:18:25](https://pagure.io/SSSD/sssd/issue/2010#636893)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
